codelabs/Getting Started with gRPC-Go/start_here/routeguide/route_guide.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 gRPC authors.
```

Comment:
For something that the codelab user is writing themselves, I don't think that the copyright makes sense.

Reply:
the dont makes sense, makes sense to me :). Updated